test/unit/vl-map-layer-style.test.html

```diff
@@ -38,31 +58,64 @@
       test('de style wordt goed geconfigureerd met default properties', () => {
         const element = fixture('vl-map-layer-style-fixture');
         const vlMapLayerStyleElement = element.querySelector('vl-map-layer-style');
-        assert.equal(vlMapLayerStyleElement.color, 'rgba(2, 85, 204, 1)');
+        assert.equal(vlMapLayerStyleElement.color, 'rgba(2, 85, 204, 0.8)');
         assert.equal(vlMapLayerStyleElement.borderColor, 'rgba(2, 85, 204, 1)');
         assert.equal(vlMapLayerStyleElement.borderSize, 1);
-        assert.equal(vlMapLayerStyleElement.textColor, '#FFF');
+        assert.equal(vlMapLayerStyleElement.textBackgroundColor, 'rgba(0, 0, 0, 0)');
+        assert.equal(vlMapLayerStyleElement.textBorderColor, 'rgba(255, 255, 255, 1)');
+        assert.equal(vlMapLayerStyleElement.textBorderSize, 1);
+        assert.equal(vlMapLayerStyleElement.textColor, 'rgba(0, 0, 0, 1)');
+        assert.equal(vlMapLayerStyleElement.textFeatureAttributeName, null);
         assert.equal(vlMapLayerStyleElement.textOffsetX, 0);
         assert.equal(vlMapLayerStyleElement.textOffsetY, 0);
-        const style = vlMapLayerStyleElement.style;
-        assert.equal(style.getFill().getColor(), 'rgba(2, 85, 204, 1)');
+        assert.equal(vlMapLayerStyleElement.textSize, '10px');
+        const style = vlMapLayerStyleElement.style();
+        assert.equal(style.getFill().getColor(), 'rgba(2, 85, 204, 0.8)');
         assert.equal(style.getStroke().getColor(), 'rgba(2, 85, 204, 1)');
         assert.equal(style.getStroke().getWidth(), 1);
       });
 
+      test('als er een featureAttributeName gedefinieerd is, zal de tekst stijl ook beschikbaar zijn', () => {
+        const element = fixture('vl-map-layer-style-met-feature-attribute-name-fixture');
+        const vlMapLayerStyleElement = element.querySelector('vl-map-layer-style');
+        assert.equal(vlMapLayerStyleElement.textFeatureAttributeName, 'label');
+        const style = vlMapLayerStyleElement.style({get: (property) => property + 1});
+        assert.equal(style.getText().getFill().getColor(), 'rgba(0, 0, 0, 1)');
+        assert.equal(style.getText().getOffsetX(), '0');
+        assert.equal(style.getText().getOffsetY(), '0');
+        assert.equal(style.getText().getFont(), '10px "Flanders Art Sans",sans-serif');
+        assert.equal(style.getText().getBackgroundFill().getColor(), 'rgba(0, 0, 0, 0)');
+        assert.equal(style.getText().getStroke().getColor(), 'rgba(255, 255, 255, 1)');
+        assert.equal(style.getText().getStroke().getWidth(), 1);
+        assert.equal(style.getText().getText(), 'label1');
+      });
```

Comment:
Test uitbreiden om aan te tonen dat wanneer `featureAttributeName` niet gedefinieerd is, de tekst stijl niet beschikbaar is?

Reply:
ok